blurring_as_a_service/move_data_pipeline/submit_move_pipeline.py

```diff
@@ -48,29 +50,26 @@ def move_files_pipeline():
 
 
 def main():
-    inference_pipeline_job = move_files_pipeline()
+    aml_interface = AMLInterface()
 
+    # Access the workspace details
+    workspace_name = aml_interface.get_workspace_name()
```

Comment:
workspace_name = aml_interface.get_workspace_name().result()
  and let's remove the "*_actual" variables